pyproject.toml

```diff
@@ -14,7 +14,7 @@ keywords = [
 license = {text = "MIT"}
 maintainers = [{name = "Martin Thoma", email = "info@martin-thoma.de"}]
 authors = [{name = "Martin Thoma", email = "info@martin-thoma.de"}]
-requires-python = ">=3.6.1"
+requires-python = ">=3.8"
```

Comment:
What was the reason for dropping Python 3.6?

Reply:
Same.

Reply:
This actually prevents people from installing it in older Python versions.

While I do see your point in not testing for newer versions, I don't want to enforce it. My suggestion would be:

* CI + Trophe classifiers for all Python versions that are still officially supported
* `requires-python` only makes the absolute necessary restrictions (e.g. I know that it doesn't work for Python 3.5 or earlier due to type annotations)

For the versions that are in between those two it would be a "your mileage may vary" situation. If people report that it doesn't work anymore I would increase the `requires-python`, but I would not enforce it when it's not necessary.

Reply:
```suggestion
requires-python = ">=3.6.1"
```

Reply:
Revert python-requires